addons/web/static/src/views/calendar/calendar_renderer.scss

```diff
@@ -82,6 +82,10 @@
             border-color: transparent;
             border-bottom-color: $border-color;
             padding: 3px 0 5px;
+
+            .fc-week-number {
+                padding-left: $table-cell-padding-x-sm;
```

Comment:
> [EDI] Calendar: misalignment in the header due to missing padding on week number https://tinyurl.com/2zzscutb

I can't replicate this issue, could you provide more context?
I'm not against the change btw since it improves the visual result of the label itself (not stick to the side), but I would ensure that the issue you faced is solved